net/nginx/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=          nginx
+PLUGIN_VERSION=       0.1
+PLUGIN_COMMENT=       nginx proxy and webserver
```

Comment:
Let's try to get this consistent for labels: **Nginx** HTTP server and reverse proxy

Reply:
done